taxcalc/tests/reforms.json

```diff
@@ -575,6 +575,14 @@
       "output_type": "iitax",
       "compare_with": {},
       "expected": "Tax-Calculator,-7.9,-7.7,-7.3,-7.2"
-  }
+  },
 
+      "58": {
+      "start_year": 2017,
+      "value": {"_cpi_offset": [-0.0025]},
+      "name": "Stack dependent credit before CTC",
+      "output_type": "iitax",
+      "compare_with": {},
+      "expected": "Tax-Calculator,0.0,2.2,4.6,7.2"
```

Comment:
@codykallen, notice how the cpi_offset reform doesn't change results in the year it is implemented. In the TCJA reform files, do you think we should implement it in 2016?

Reply:
> Do you mean 2017 (so that it first takes effect in 2018)? 

I was looking at this: https://waysandmeansforms.house.gov/uploadedfiles/bill_text.pdf
![image](https://user-images.githubusercontent.com/8301092/33001124-efc2d89e-cd79-11e7-8d88-8541f5178602.png)